### ISSUE TITLE
Sync Batch norm not used?

### ISSUE THREAD

**Issue by dragen1860** [daa233/generative-inpainting-pytorch#34]
Dear author:
  Thanks for your re-implementation, it's helpful!
  When I tried to train with multi gpus, I noticed you have not convert the batch norm to synced batch norm. I wonder if you realized it or not？ Thank you.

**Comment by daa233** [comment]
@dragen1860 Thanks for your attention!

The synced batch norm is not used in this repo. The original tensorflow implementation of deep fill v1 uses only 1 gpu to train the model. I extended this repo to support multiple gpus but not synced batch norm. You are welcomed to contribute the code!
